src/CampaignsAdmin/report/locationCard.css

```diff
@@ -0,0 +1,34 @@
+.location-card {
+  flex: 0 1 auto;
+  /* width: 400px; */
+  background-color: #eff2f5;
+  margin: 10px;
+  border: 2px solid #173057;
+  }
+.location-card:hover{
+  box-shadow: 10px 10px 20px rgba(0, 0, 0, 0.2);
+}
+
+.location-card .card-image{
```

Comment:
Also, I don't know if we're doing any pre-processing via babel/webpack, but we want to end up with something like this eventually: 

```css
.location-card {
  .card-info {
    background-color: #eff2f5;
    padding: 15px;
  }

  .card-image {
    box-shadow: 0px 6px 5px rgb(0 0 0 / 10%)
  }

  .stats {
    height: 30px;
    background-color: #eff2f5;
    padding: 5px;
  }

  h3 {
    color: #173057;
    font-size: 1.3em;
    text-transform: uppercase;
    margin-bottom: 0;
  }

  .ant-ribbon-text {
    color: #eff2f5;
  }
}

```